gbasis/evals/basis_eval.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 
 
-class Eval(BaseOneIndex):
+class BasisEval(BaseOneIndex):
```

Comment:
This is so much better

Reply:
Thanks for your comment @wilhadams. See 26ae9aa